src/components/Signin/Signin.jsx

```diff
@@ -0,0 +1,78 @@
+import Image from "next/image"
+import { BsFacebook, BsGoogle, BsTwitter } from "react-icons/bs"
+
+import styles from "./Signin.module.css"
+
+import Button from "../button"
+import Input from "../input"
+function Signin() {
+  return (
+    <>
+      <div className={`flex justify-center  md:flex-row  bg-[#f1f6fa] `}>
+        <div className={` ${styles.handbox_background}   w-3/5 `}>
+          <div className=' p-20 '>
+            <Image
+              src='/images/hands_box.png'
+              alt='handbox'
+              width={500}
+              height={500}
+            />
+          </div>
+        </div>
+        <div className='container m-auto flex w-5/6 flex-col items-center'>
```

Comment:
When you notice that some elements having the same exact tailwind classes, why don't you extract them and put them in a variable above here, or in the file you created (the module CSS file), and then you just put it inside the className attribute, instead of putting all the classes repeatedly, more than one time.